references/detection/train.py

```diff
@@ -33,6 +33,12 @@
 from group_by_aspect_ratio import GroupedBatchSampler, create_aspect_ratio_groups
 
 
+try:
+    from torchvision.prototype import models as PM
```

Comment:
Temporarily adding support of prototype on the detection reference script similar to all other areas, so we can test the results.